scripts/filter_markets.py

```
@@ -346,11 +347,47 @@ def order_score(v: float, s: float, size: float, min_size: float) -> float:
+    if ok:
+        return None
+    return {
+        "source": source, "eligible": False, "reject_reason": reason,
+        "queue_minutes": minutes,
+        "cid": m.get("condition_id"),
+        "title": m.get("question", "")[:90],
+        "slug": m.get("market_slug", ""),
+    }
+
+
 def evaluate(session: requests.Session, rate: float, m: dict,
```

Ruff (0.16.2)

[warning] Too many return statements (11 > 6)

(PLR0911)